wagtailio/settings/base.py

```diff
@@ -382,9 +384,13 @@
 
 WAGTAILIMAGES_IMAGE_MODEL = "images.WagtailioImage"
 
+WAGTAILIMAGES_FORMAT_CONVERSIONS = {
+    'gif': 'gif',
```

Comment:
Do we need this? Wagtail will automatically convert non-animated GIFs to PNG, but leave animated GIFs in GIF format.

Reply:
Right, it appears not. I guess I added this in a misguided debugging attempt.

Reply:
Removed in 42105e7355f8c8288570ecc3e8cb87e2faf320c8